jenkins/ci.suse.de/cloud-mkcloud9-suse.yaml

```diff
@@ -0,0 +1,58 @@
+- job:
+    name: 'cloud-mkcloud9-suse-x86_64'
+    project-type: multijob
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - multijob:
+          name: 'Standard Gate Checks (non-HA/OVS+Linuxbridge)'
+          condition: SUCCESSFUL
+          projects:
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                cloudsource=susecloud
+                TESTHEAD=
```

Comment:
Maybe add a FIXME comment in there so we remember switching to TESTHEAD=1 when the staging project is ready.

Reply:
no, it is intentionally unset (TESTHEAD has no meaning for cloudsource=susecloudX other than including "test updates" which we don't want here..)